File: CafeTier2/src/main/java/group7/Restful/entity/Booking.java
package group7.Restful.entity;

import lombok.Data;
import java.util.UUID;
@Data
public class Booking {
    private UUID UserId;
    private UUID EventId;
    private String CreationDate;
    private int NumberOfPeople;
    private UUID id;
    private String FirstName;
    private String LastName;

    public Booking() {
    }

    public Booking(UUID UserId, UUID EventId, String CreationDate, int NumberOfPeople){
        this.UserId = UserId;
        this.EventId = EventId;
        this.CreationDate = CreationDate;
        this.NumberOfPeople = NumberOfPeople;
    }
}

File: DataTier1/Data/Models/Booking.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Data.Models
{
    public class Booking : BaseEntity
    {
        [Required]
        public User User { get; set; }

        [Required]
        public Guid UserId { get; set; }
        [Required]
        public Event Event { get; set; }
        [Required]
        public Guid EventId { get; set; }
        [Required]
        public DateTime CreationDate { get; set; }

        [Required]
        public int NumberOfPeople { get; set; }


    }
}
